changelog.md

```diff
@@ -6,6 +6,40 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### 0.41.0
+
+### Added
+
+* `state init` is now a stable command, meaning you no longer need to opt-in to
+  unstable commands to use it.
+* Signing up for a new account now opens the account creation page in your
+  browser.
+* `state shell` can now detect currently active subshells preventing nested
+  shells from being created.
+* The State Tool will now print a message when starting a subshell, indicating
+  the project that is being used.
+* Wildcard version matching is now supported for `state install`.
+* Full or partial version matching is now supported for language versions when
+  using `state init`.
+* Added messaging on the potentially disruptive nature of editing or moving a project.
+* Users can now check out a project without cloning the associated git
+  repository.
+
+### Fixed
+
+* Several error message improvements for errors arising from unauthenticated actions.
+* Fixed failed runtime installations due to parsing errors.
+* No longer including unneccessary artifacts in runtime installations.
```

Comment:
These don't sound very user facing. Try to address what was fixed for the user. Not what was changed under the hood to facilitate the fix.